reaper.go

```diff
@@ -119,34 +116,31 @@ func (r *reaper) reap() error {
 // start starts the exec command and registers the process to the reaper.
 // This function is a helper for exec.Cmd.Start() since this needs to be
 // in sync with exec.Cmd.Wait().
-func (r *reaper) start(c *exec.Cmd) error {
+func (r *reaper) start(c *exec.Cmd) (chan int, error) {
```

Comment:
returns the receiving channel `<-chan int`

Reply:
PR updated.